javascript/javascript2/week1/animal.js

```diff
@@ -0,0 +1,107 @@
+
+
+// Spirit animal name generator: 
+
+const spiritAnimals=['ant', 'alligator', 'anaconda', 'albatross', 'antelope', 'armadillo', 'hawk', 'eagle', 'hummingbird', 'owl', 'butterfly']; 
+ const yourSpiritAnimal= spiritAnimals[Math.floor(Math.random()*spiritAnimals.length)];
+  
+console.log(spiritAnimals);
+  let name; 
+
+  const text= document.getElementById('username'); 
+  const button=document.getElementById('button'); 
+  const output=document.getElementById('output'); 
+  
```

Comment:
What if user did not mentioned the name? You can add an if statement for checking it, because now it can generate  without mentioning a name 🤔